modules/azure/aks-global/variables.tf

```diff
@@ -84,15 +84,3 @@ variable "aks_group_name_prefix" {
   type        = string
   default     = "aks"
 }
-
-variable "azad_kube_proxy_config" {
-  description = "Azure AD Kubernetes Proxy configuration"
-  type = object({
-    cluster_name_prefix = string
-    proxy_url_override  = string
-  })
-  default = {
-    cluster_name_prefix = "aks"
-    proxy_url_override  = ""
-  }
-}
```

Comment:
Might it be worth of actually creating a aks-global and store this config in that module?